pkg/cli/subcommands/clusterchecks/command.go

```diff
@@ -85,6 +88,23 @@ func MakeCommand(globalParamsGetter func() GlobalParams) *cobra.Command {
 
 	cmd.AddCommand(rebalanceCmd)
 
+	isolateCmd := &cobra.Command{
+		Use:   "isolate --checkID <checkID>",
```

Comment:
I think `--checkID <checkID>` should not be included here because `--help` shows this:
```
Usage:
  agent clusterchecks isolate --checkID <checkID> [flags]

Flags:
      --checkID string   the check ID to isolate
  -h, --help             help for isolate
  ```
  
  So `[flags]` already includes the checkID.